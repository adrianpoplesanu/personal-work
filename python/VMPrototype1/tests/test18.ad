if ((if (false) { 10 })) { 10 } else { 20 }