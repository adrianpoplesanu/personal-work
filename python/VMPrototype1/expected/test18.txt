ASTProgram
ASTExpressionStatement\[ASTPrefixExpression <!> \[ASTIfExpression\]\]
\[7, 14, 0, 10, 0, 0, 0, 15, 0, 11, 16, 12, 5\]
0000 OpFalse
0001 OpJumpNotTruthy 10
0004 OpConstant 0
0007 OpJump 11
0010 OpNull
0011 OpBang
0012 OpPop
true
